djangocms_pageadmin/cms_toolbars.py

```diff
@@ -0,0 +1,57 @@
+from collections import OrderedDict
+
+from django.apps import apps
+from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
+
+
+from cms.cms_toolbars import PageToolbar
+from cms.toolbar.items import ButtonList
+from cms.toolbar_pool import toolbar_pool
+from cms.utils import get_language_from_request, page_permissions
+from cms.utils.page_permissions import (
+    user_can_change_page,
+    user_can_delete_page,
+)
+from cms.utils.urlutils import add_url_parameters, admin_reverse
+
+from djangocms_versioning.models import Version
+
+
+# Identifiers for search
+ADMIN_MENU_IDENTIFIER = 'admin-menu'
+VERSIONING_MENU_IDENTIFIER = 'version'
+
+
+class PageAdminToolbar(PageToolbar):
+
+    def change_admin_menu(self):
+        """
+        The standard Pages menu for the PageContent model as provided by the CMS returns a selected item.
+        This works because that view has no filters. PageAdmin has filters, so this causes unwanted filtering.
+        Thus replace the PageToolbar with to remove the filter parameters from the URL
+        """
+        super(PageAdminToolbar, self).change_admin_menu()
+        menu = self.toolbar.get_menu(ADMIN_MENU_IDENTIFIER)
+        if menu.items[0].name == 'Pages...':
```

Comment:
Think of a different test here, this won't work if current request has a different language

Reply:
I can use the URL instead of the name. Is there any chance that the URL for a menu item will be `None`? If so then I need to wrap it in a `try catch`.